stb_vorbis.c

```diff
@@ -4427,7 +4479,7 @@ static int vorbis_search_for_page_pushdata(vorb *f, uint8 *data, int data_len)
             f->next_seg = -1;       // start a new page
             f->current_loc = f->scan[i].sample_loc; // set the current sample location
                                     // to the amount we'd have decoded had we decoded this page
-            f->current_loc_valid = f->current_loc != ~0U;
+            f->current_loc_valid = f->current_loc != ~(int64)0U;
```

Comment:
Like above, lose the `U` suffix. The constant can simply be `~((int64)0)`

Reply:
// better say "remove `U` suffix", the "lose" in this case sounds absolutely weird to me... "Hey, dude, lose your wallet on the bus seat! I wanna steal it!"

Done

Reply:
> // better say "remove `U` suffix", the "lose" in this case sounds absolutely weird to me... "Hey, dude, lose your wallet on the bus seat! I wanna steal it!"

Eh?? It's a common expression, never thought about it that way.

Reply:
“Lose” for “intentionally get rid of,” especially as a suggestion or command, is a common US English slang idiom since at least the mid-20th century. Presumably it’s not quite universal among global English speakers.

Reply:
Possibly, I haven't (yet) followed the code thoroughly.